lib/puppet_x/msutter/templates/invoke_dsc_resource.erb

```diff
@@ -42,12 +42,13 @@ $invokeParams = @{
 <% dsc_parameters.each do |p| -%>
     <%= p.name.to_s.gsub(/^dsc_/,'') %> = <%= format_dsc_value(p.value) %>
 <% end -%>
-  }
-<% if resource[:dscmeta_module_name].casecmp('PSDesiredStateConfiguration') == 0 -%>
```

Comment:
I'm guessing this for some reason wasn't always catching the name for PSDesiredStateConfiguration?